File: Basic Coding Problems/01 Fundamentals/006 Write a program to enter the radius of a circle and find its diameter, circumference and area/C++/circle.cpp
#include <iostream>

using namespace std;

int main()
{
    int radius;
    float diameter, circumference, area;

    cout << "Enter the radius of the circle: ";
    cin >> radius;

    diameter = 2 * radius;
    circumference = 2 * 3.14 * radius;
    area = 3.14 * radius * radius;

    cout << "The diameter of the circle is: " << diameter << endl;
    cout << "The circumference of the circle is: " << circumference << endl;
    cout << "The area of the circle is: " << area << endl;

    return 0;
}

File: Basic Coding Problems/01 Fundamentals/006 Write a program to enter the radius of a circle and find its diameter, circumference and area/Java/circle.java
import java.util.Scanner;

public class circle {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the radius of the circle: ");
        float radius = scanner.nextFloat();

        float diameter = 2 * radius;
        float circumference = 2 * (float) Math.PI * radius;
        float area = (float) Math.PI * radius * radius;

        System.out.println("The diameter of the circle is: " + diameter);
        System.out.println("The circumference of the circle is: " + circumference);
        System.out.println("The area of the circle is: " + area);
    }
}